wdiff.c: In function ‘complete_input_program’:
wdiff.c:916: warning: implicit declaration of function ‘wait’

--- wdiff.c
+++ wdiff.c
@@ -45,6 +45,7 @@
 # include <stdlib.h>
 #endif
 
+#include <sys/wait.h>
 #include <ctype.h>
 #include <stdio.h>
 
